tests/dofs/interpolate_q_system_mask_02.output

```diff
@@ -0,0 +1,93 @@
+
+DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)], Mask 1, P_0, rel. error=0
+DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)], Mask 2, P_0, rel. error=0
+DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)], Mask 1, P_1, rel. error=0
+DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)], Mask 2, P_1, rel. error=0
+DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)], Mask 1, P_2, rel. error=0.000425
+DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)], Mask 2, P_2, rel. error=0.000425
+DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)], Mask 1, P_3, rel. error=0.00125
+DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)], Mask 2, P_3, rel. error=0.00125
```

Comment:
The test still includes the assertion from interpolate_q_system_01, and I checked these against the unmasked output (interpolate_q_system_01, interpolate_q_system_02) for where the nonzero values started.